content/blog/usemodels-0-0-1/index.Rmd

```diff
@@ -0,0 +1,106 @@
+---
+output: hugodown::hugo_document
+
+slug: usemodels-0-0-1
+title: usemodels 0-0-1
+date: 2020-09-29
+author: Max Kuhn
+description: >
+    The new usemodels R package is a helpful way to automatically generate 
+    tidymodels code. 
+
+photo:
+  url: https://unsplash.com/photos/0TH1H1rq_eY
+  author: Neven Krcmarek
+
+categories: [package] 
+tags: [tidymodels,parsnip,recipes,tune]
+---
+
+```{r startup, include = FALSE}
+library(tidymodels)
+library(usemodels)
+options(width = 90)
+```
+
+We're very excited to announce the first release of the [usemodels](https://usemodels.tidymodels.org/) package. usemodels creates templates for tidymodels analyses so you don't have to write as much new code. 
+
+You can install it from CRAN with:
+
+```{r, eval = FALSE}
+install.packages("usemodels")
+```
+
+This blog post will show how to use the package. 
+
+Let's start with something simple: creating a glmnet linear regression model for the `mtcars` data using tidymodels. This model is usually tuned over the amount and type of regularization. In tidymodels, there are a few intermediate steps for a glmnet model: 
```

Comment:
[](url)
```suggestion
Let's start with creating a glmnet linear regression model for the `mtcars` data using tidymodels. This model is usually tuned over the amount and type of regularization. In tidymodels, there are a few intermediate steps for a glmnet model: 
```

Reply:
With @juliasilge's change, I think the with should become "by" (Let's start by creating a glmnet linear-regression model"